packages/experiments/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "target": "es5",
     "module": "commonjs",
     "jsx": "react",
+    // "jsxFactory": "Slot",
```

Comment:
is this still necessary?

Reply:
Nope, I still have quite a bit of cleanup and TODOs to resolve (in description.) I wanted to start getting feedback on the typings and approach as highlighted in the PR description.